src/routers/history.js

```diff
@@ -0,0 +1,24 @@
+const HistoryService = require('../services/history')
+const { getHistoryByTargetIDSchema } = require('../schemas/services/history')
+
+module.exports = async (server, opts) => {
+  const { mongol } = opts
+  const historyService = new HistoryService(mongol)
+
+  server.get('/history/targets/:targetID', { schema: getHistoryByTargetIDSchema }, async (req, res) => {
+    try {
+      const { targetID } = req.params
+      const { limit, after, before } = req.query
+      const result = await historyService.getHistoryByTargetID(
+        targetID,
+        limit,
+        after,
+        before,
+      )
+      res.code(200).send(result)
+    } catch (err) {
+      req.log.error(err.message)
```

Comment:
```suggestion
      server.log.error(err.message)
```
Mình ko chắc lỗi này ở đâu ra mà, nên cứ để server cho lành.